ui/components/component-library/base-avatar/base-avatar.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import Box from '../../ui/box/box';
+
+function BaseAvatar(props) {
+  const { size } = props;
+  return (
+    <Box
+      className={classnames('base-avatar', ` --size-${size}`)}
+      data-testid="base-avatar "
+    ></Box>
```

Comment:
This component will need to accept children
```suggestion
    <Box
      className={classnames('base-avatar', ` --size-${size}`)}
      data-testid="base-avatar "
    >{children}</Box>
```